fts/tests.py

```diff
@@ -449,9 +449,11 @@ def mockflatten(input_name, output_name, *args, **kwargs):
     body_text = browser.find_element_by_tag_name('body').text
     if len(warning_texts) == 0:
         assert 'Conversion Warnings' not in body_text
+        assert 'conversion errors' not in body_text
     else:
         assert warning_texts[0] in body_text
         assert 'Conversion Warnings' in body_text
+        assert 'conversion errors' not in body_text
```

Comment:
These are useful, but it might be good to have both of them be case insensitive, e.g.:
```
         assert 'Conversion Warnings' in body_text.lower()
         assert 'conversion errors' not in body_text.lower()
```